tools/buildsteps/windows/BuildSetup.bat

```diff
@@ -13,7 +13,7 @@ SET sed_exe=%msys_dir%\usr\bin\sed.exe
 
 REM read the version values from version.txt
 FOR /f %%i IN ('%awk_exe% "/APP_NAME/ {print $2}" %base_dir%\version.txt') DO SET APP_NAME=%%i
-FOR /f %%i IN ('%awk_exe% "/COMPANY_NAME/ {$1=""; print $0}" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i
+FOR /f "tokens=2*" %%i IN ('findstr "COMPANY_NAME" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i %%j
```

Comment:
maybe sed -n '/COMPANY_NAME/ s/COMPANY_NAME *//p'

Reply:
thanks. this much simpler